test/spec/modules/openxBidAdapter_spec.js

```diff
@@ -1225,14 +1143,16 @@ describe('OpenxAdapter', function () {
     });
 
     it('should send a bc parameter', function () {
-      const request = spec.buildRequests(bidRequestsWithMediaTypes);
+      const request = spec.buildRequests(bidRequestsWithMediaTypes, mockBidderRequest);
       const dataParams = request[0].data;
 
       expect(dataParams.bc).to.have.string('hb_pb');
     });
 
     describe('when using the video param', function () {
       let videoBidRequest;
+      let mockBidderRequest = {refererInfo: {}};
+
```

Comment:
It seems that there is two blank lines here and eslint is not happy about it.

Reply:
fixed